WordPress/twentytwentyfive
# Add patterns for the Photo blog

**Description**

Adds <del>templates and</del> patterns for the photo blog.
Closes https://github.com/WordPress/twentytwentyfive/issues/41

[Figma link](https://www.figma.com/design/dzGCSntVch4EQdVERTqyVK/Twenty-Twenty-Five?node-id=3155-1263&t=4ZvvKDNEjcR8g2uv-4)

**Testing Instructions**
 On your WordPress test installation, create enough posts for the query pagination and next and previous posts to work:
 Create posts with and without featured images, with and without tags, and with no or several categories.
 
 Create two pages, one with and one without featured image.
 
Apply the PR and activate the theme.
Activate the Gutenberg plugin.

Go to Appearance > Editor and select the home template.
Open the Design panel and switch to "photo blog home template" (this needs a better shorter name).
Confirm if the design matches the design in Figma.
Save and view the the front, and confirm if it looks correct on the front.

Now repeat the test for the other templates: Archive, search, single post, page.

 
**Screenshots**


(out of date screenshots deleted)
